server/container/guice/configuration/src/main/java/org/apache/james/utils/PropertiesProvider.java

```diff
@@ -35,15 +35,17 @@
 public class PropertiesProvider {
 
     private final FileSystem fileSystem;
+    private final String configurationPrefix;
 
     @Inject
-    public PropertiesProvider(FileSystem fileSystem) {
+    public PropertiesProvider(FileSystem fileSystem, org.apache.james.server.core.configuration.Configuration configuration) {
```

Comment:
is this changeset used anywhere ?

Reply:
Yes, in every guice products, that's why I added the bind of the _Configuration_ in _CommonServicesModule_.

Reply:
It's the same pattern for _FileConfigurationProvider_